test/models/lookup_key_test.rb

```diff
@@ -420,4 +420,30 @@ def setup
       assert_valid lookup_key
     end
   end
+
+  test "can update lookup key with default_value without audit error" do
+    loc1 = taxonomies(:location1)
+    loc2 = taxonomies(:location2)
+    location_list = [loc1, loc2]
+    puppetclass1 = FactoryBot.create(:puppetclass, :with_auditing)
+    env1 = FactoryBot.create(:environment, :name => 'ENV1')
+    env2 = FactoryBot.create(:environment, :name => 'ENV2')
+    env3 = FactoryBot.create(:environment, :name => 'ENV3')
+    [env1, env2, env3].each do |e|
```

Comment:
similar change, since there is no significance of the `ENV1`, `ENV2` and `ENV3` here we can do something like:
```
FactoryBot.create_list(:environment, 3)
```
to refer to its implementation you can refer to https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#building-or-creating-multiple-records

Reply:
:+1:  thanks, it looks much cleaner than previous :)